src/more-infos/more-info-media_player.js

```diff
@@ -70,12 +70,34 @@ export default new Polymer({
       value: false,
     },
 
+    supportsVolumeButtons: {
+      type: Boolean,
+      value: false,
+    },
+
+    hasMediaControl: {
+      type: Boolean,
+      value: false,
+    },
+
+  },
+
+  attached() {
+    // This is required to bind a mousedown event in all browsers
+    const _this = this;
+    this.$.volumeUp.onmousedown = function onVolumeUpDown() {_this.handleVolumeUp();};
```

Comment:
In my opinion, this is a crucial feature for volume up and down buttons. Volume buttons become impossible to use if you have to click it for each increment you want to move.

If I want to move the volume from 20 to 35 (a very regular request), clicking the increment button 15 times gets crazy. Button mashing is far from accurate and is quite fatiguing.

I can definitely change the approach. JavaScript is not my strong suit. I bound the events in the JavaScript because, when I used the preferred method, I was having trouble in Safari. I can try the preferred method again. I may have been doing something else wrong.